### PR TITLE
[plugin-mobile-app] Add step to clear a field and keep keyboard open

### PR DESCRIPTION
**Pull request by VerkhovtsovPavel**
Closes https://github.com/vividus-framework/vividus/issues/2617

### CODE REVIEW

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java [hunk 1]
```diff
@@ -94,10 +94,19 @@ public void typeText(WebElement element, String text)
      * </ol>
      * @param element element to clear, must not be {@code null}
      */
+    public void clearTextAndHide(WebElement element)
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/KeyboardActions.java:97]: it makes sense to have 2 methods in `KeyboardActions`:
- `clearText`
- `hideKeyboard`

any combinations of these methods invocations should be done in steps

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/KeyboardSteps.java [hunk 2]
```diff
@@ -109,6 +110,16 @@ public void typeTextInField(String text, Locator locator)
                 .ifPresent(e -> keyboardActions.typeTextAndHide(e, text));
     }
 
+    /**
+     * Clear a field located by the <b>locator</b>
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/KeyboardSteps.java:114]: need to document explicitly that this step closes keyboard

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — COMMENTED** [review]
-

**Review by valfirst — APPROVED** [review]

**Review by ikalinin1 — APPROVED** [review]

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/KeyboardSteps.java [hunk 3]
```diff
@@ -106,7 +107,20 @@ public void typeTextInFieldAndKeepKeyboard(String text, Locator locator)
...
+                    keyboardActions.typeText(e, text);
+                    keyboardActions.hideKeyboard(e);
+                });
+    }
+
+    /**
+     * Clear a field located by the <b>locator</b>
+     * @param locator locator to find a field
+     */
+    @When("I clear field located `$locator` and keep keyboard opened")
```
Comment (uarlouski) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/steps/KeyboardSteps.java:120]: hm I think it should be "open" => https://english.stackexchange.com/questions/141547/opened-vs-open

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 4]
```diff
@@ -681,6 +681,23 @@ When I clear field located `accessibilityId(username)`
...
+----
+When I clear field located `$locator` and keep keyboard opened
+----
+
+* `$locator` - <<_locator>>.
+
+.Clear text and keep keyboard opened
+[source,gherkin]
+----
+When I clear field located `accessibilityId(username)` and keep keyboard opened
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:698]:
```suggestion
When I clear field located `accessibilityId(username)` and keep keyboard open
```

File: vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story [hunk 5]
```diff
@@ -155,6 +155,18 @@ When I tap on element located `accessibilityId(Return)`
...
+    @requirementId 2617
+    @targetPlatform ios
+!-- Typing on android emulator doesn't shows a keyboard.
+When I clear field located `accessibilityId(nameInput)` and keep keyboard opened
+Then number of elements found by `xpath(<nameDisplayXpath>)` is equal to `1`
+Then number of elements found by `<keyboardLocator>` is equal to `1`
+When I tap on element located `accessibilityId(Return)`
+Then number of elements found by `<keyboardLocator>` is equal to `0`
+
+
```
Comment (ikalinin1) [inline vividus-tests/src/main/resources/story/system/mobile_app/MobileAppStepsTests.story:169]:
```suggestion
Scenario: Verify step: 'When I clear field located `$locator` and keep keyboard open'
Meta:
    @requirementId 2617
    @targetPlatform ios
!-- Typing on android emulator doesn't shows a keyboard.
When I clear field located `accessibilityId(nameInput)` and keep keyboard open
Then number of elements found by `xpath(<nameDisplayXpath>)` is equal to `1`
Then number of elements found by `<keyboardLocator>` is equal to `1`
When I tap on element located `accessibilityId(Return)`
Then number of elements found by `<keyboardLocator>` is equal to `0`


```

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 6]
```diff
@@ -681,6 +681,23 @@ When I clear field located `accessibilityId(username)`
 ----
 
 
+=== Clear field and keep keyboard opened
```
Comment (ikalinin1) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:684]:
```suggestion
=== Clear field and keep keyboard open
```

File: docs/modules/plugins/pages/plugin-mobile-app.adoc [hunk 7]
```diff
@@ -681,6 +681,23 @@ When I clear field located `accessibilityId(username)`
...
+Only clears the field located by the locator. The keyboard hiding is skipped.
+
+[source,gherkin]
+----
+When I clear field located `$locator` and keep keyboard open
+----
+
+* `$locator` - <<_locator>>.
+
+.Clear text and keep keyboard opened
```
Comment (valfirst) [inline docs/modules/plugins/pages/plugin-mobile-app.adoc:695]:
```suggestion
.Clear text and keep keyboard open
```

**Review by uarlouski — APPROVED** [review]

**Pull request merged**